pyproject.toml

```diff
@@ -303,3 +304,57 @@ exclude_lines = [
     'if __name__ == .__main__.:',
 ]
 ignore_errors = true
+
+[tool.pyright]
+typeCheckingMode = "strict"
+include = ["sphinx", "utils", "tests", "doc/conf.py"]
```

Comment:
```suggestion
include = [
    "doc/conf.py",
    "utils", 
    "sphinx",
    "tests", 
]
```

Reply:
FWIW in my own projects I use [`pyproject-fmt`](https://github.com/tox-dev/pyproject-fmt) to avoid thinking about this formatting.